.github/workflows/Promote Packages.yml

```diff
@@ -0,0 +1,50 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: Promote Packages
+
+on:
+  workflow_dispatch:
+    inputs:
+      promote_packages_view:
+        description: ''select promote packages view for Azure Artifacts'
+        required: true
+        default: 'Prerelease'
+        type: choice
+        options:
+        - Prerelease
+        - Release
+        - Local
```

Comment:
If we need to keep the Local option for testing then that is fine. But I tested in my own fork of the repo and only two options are required for the input to ensure its a drop down, as seen here 
![image](https://user-images.githubusercontent.com/1013553/180883339-16cf1cab-b7c5-4517-ad5a-5db63bff895f.png)

Reply:
I removed local